src/tools/pontoon/ui.md

```diff
@@ -4,115 +4,123 @@
+Note that some profile menu items are only available to users with specific [permissions](users.md#user-roles).
 
-![Profile menu](../../assets/images/pontoon/ui/profile_menu.png)
+### String list and filters
 
-Note that some profile menu items are restricted by [user rights](users.md#user-roles).
+The sidebar displays the list of strings in the current project resource. For each string there is:
+* A colored square that identifies the string status (i.e. *Missing*, *Translated*, etc.).
+* The source string.
+* The approved translation or the most recent suggestion if available.
 
-### Sidebar
-
-The sidebar displays the list of strings in the current project resource. Each string is displayed with the string status (i.e. *Missing*, *Translated*, etc.) identified by a colored square, the source string, and the approved translation or the most recent suggestion if available.
-
-![Sidebar](../../assets/images/pontoon/ui/sidebar.png)
+![Sidebar](../../assets/images/pontoon/ui/sidebar.png "Screenshot of the sidebar, with a list of strings showing the different string statuses")
 
 Color legend:
 * **<span style="color: #7bc876;">green</span>**: string is translated.
-* **<span style="color: #fed271;">yellow</span>**: string is fuzzy.
-* **<span style="color: #ffa10f;">orange</span>**: string has warnings.
-* **<span style="color: #f36;">red</span>**: string has errors.
-* **<span style="color: #4d5967;">gray</span>**: string is missing.
+* **<span style="color: #c0ff00;">light-green</span>**: string is pretranslated.
+* **<span style="color: #ffa10f;">orange</span>**: translation has warnings.
+* **<span style="color: #f36;">red</span>**: translation has errors.
+* **<span style="color: #4d5967;">gray</span>**: translation is missing.
+
+At the top of the sidebar, the user can access Pontoon’s [search and string filters](search_filters.md).
 
-At the top of the sidebar, the translator can access Pontoon’s [search and string filters](search_filters.md).
+When a string is selected in the sidebar, a small icon with 4 arrows is displayed near the checkbox: this can be used to show strings that surround the selected string in the [resource](glossary.md#resource), bypassing the current filter. This is often helpful to provide more context for the localization, especially when translating missing strings.
 
-Note that the sidebar toggles between the strings and the editing space when using the in-context editor.
+![Button to show sibling strings in sidebar](../../assets/images/pontoon/ui/sidebar_expand.png "Screenshot of the button to show sibling strings in sidebar")
 
 ### Translation workspace
```

Comment:
We already have `## Pontoon translation workspace` (a level up title) above, which is essentially the same. Should this instead be `Main editing space` or `Middle column` as referred to in the intro paragraph? The "translation workspace" references below should then also be updated.

Relatedly, why do we have 2 title levels at the top of the document, and then everything that follows at level 3 or 4?
```
# Understanding Pontoon user interface
## Pontoon translation workspace
```

`# Understanding Pontoon translation workspace` would be a more accurate title, and the rest of the titles could then be moved a level up.